src/main/resources/i18n/messages_de.properties

```diff
@@ -152,6 +152,7 @@ game.create.map = Karte
 game.create.title = Titel (erforderlich)
 game.create.password = Passwort (optional)
 game.create.randomMap = Zufällige Karte
+game.create.generatedMap=Generiere Karte
```

Comment:
Dont make any translations by hand. They are done at this website which name i can not remember and are think if you do it nevertheless yours gets overwritten or something similar. I just remember you should not do so.